pkg/controller/images/controller_images.go

```diff
@@ -47,6 +53,13 @@ func GetHiveClusterProvisionImagePullPolicy() corev1.PullPolicy {
 	return corev1.PullPolicy(envVarOrDefault(HiveClusterProvisionImagePullPolicyEnvVar, string(corev1.PullIfNotPresent)))
 }
 
+// GetInstallerClusterProvisionImagePullPolicy returns the policy to use when pulling the installer image
+// for the ClusterProvision pod.
+// Either the one specified in the environment variable or the hardcoded default.
+func GetInstallerClusterProvisionImagePullPolicy() corev1.PullPolicy {
+	return corev1.PullPolicy(envVarOrDefault(InstallerClusterProvisionImagePullPolicyEnvVar, string(corev1.PullIfNotPresent)))
```

Comment:
We would also need a way to get this env var down to the controller. In fact, it looks to me like the `HiveClusterProvisionImagePullPolicyEnvVar` (upon which I'm guessing you based this change) can't be used today for that same reason.